views/home.hbs

```diff
@@ -0,0 +1,23 @@
+<div id="addon">
+    <img class="logo" src="{{manifest.logo}}">
+    <h3 class="version"><i>{{manifest.version}}</i></h3>
+    <h3 class="description">{{manifest.description}}</h3>
+
+    <h2 class="gives">This add-on has:</h2>
+    <ul>
+        <li>
+            More
+            {{#each manifest.types}}
+                {{stylize this}}
+            {{/each}}
+            in Discover
+        </li>
+        <li>Watch for FREE</li>
+    </ul>
+
+    <a class="activate-link" href="stremio://{{manifest.id}}/stremioget/stremio/v1"><button name="Install">Install Add-on</button></a>
```

Comment:
this install button won't work, it used to be possible to install add-ons by their `manifest.id` but this isn't possible anymore

the way `stremio://` links work now is that you need the actual add-on manifest url, so if your manifest url is `https://myaddon.com/manifest.json`, then the stremio link for it should be `stremio://myaddon.com/manifest.json`

but this creates an issue, because we don't (and shouldn't) set the manifest url in the manifest itself (there used to be an `endpoint`, it's still in the docs but should be removed

what i'm proposing is that this button stay greyed out by default, we add a `.publishToWeb(url)` method that saves the manifest url, replaces `https:` with `stremio:` and uses it as the href of this link only when it's available

Reply:
Do you think it would be possible for it to be `runHTTPWithOptions` that automatically takes care of that? Like this:

```
const server = addonHTTPApp.listen(options.port, function() {
   var url = 'http://127.0.0.1:'+server.address().port;
   var manifestUrl = `${url}/manifest.json`;
   console.log('HTTP addon accessible at:', manifestUrl)

   // Set default logo & background if not set in the manifest
   if (!manifest.logo) manifest.logo = `${url}/static/imgs/logo.png`;
   if (!manifest.background) manifest.background = `${url}/static/imgs/background.jpg`;

   // Create a protocol url
   manifestRender = JSON.parse(JSON.stringify(manifest));
   manifestRender.protocol = manifestUrl.replace(/^(https?|ftp):\/\//, 'stremio://');

   // Render the home page
   addonHTTP.route('/').get(function (req, res) {
      res.render('home', {manifest: manifestRender});
   });

   if (cb) cb(null,  { server: server, url: url })
})
```

Reply:
well no, that's not possible..

the reason that isn't possible is because you are presuming that the add-on is local, but if someone is hosting it on a remote server and that server has a domain set too, then it's simply impossible for this to be done like this as we can't know the domain without the developer inputting it directly

so the only viable solution is to set it with a method (the proposed `.publishToWeb(url)')

Reply:
Oh ok I see, I'll do that then !